lib/views/admin_request/hidden_user_explanation/_immigration_information.text.erb

```diff
@@ -0,0 +1,30 @@
+<%= _('As it is personal correspondence which it wasn\'t appropriate to ' \
```

Comment:
I think we should make this the same string as https://github.com/mysociety/alaveteli/blob/develop/app/views/admin_request/hidden_user_explanation/_personal_information.text.erb#L1-L3 to reduce the number of translated strings. The "You will still be able to view it" part is duplicated at the end of this template.